### DISCUSSION TITLE
Noise and distortion problems with VITS training

### DISCUSSION THREAD

**Discussion by mllopartbsc** [coqui-ai/TTS#3031] [General Q&A]
Hello everyone,

We are training the VITS TTS model for the Catalan language and we've encountered that the resulting inference of the model is noisy and distorted. The model we've created is a fine-tuning from the original English VITS TTS model. Here's the Weights & Biases boards of 4 different training runs: 

https://wandb.ai/bsc-language-tech/finetune_ca_from_en

Here are two examples of noisy audios from the `finetune_en_5e-5_5e-5_gensched-September-29-2023_10+27PM-0000000` run:

https://github.com/coqui-ai/TTS/assets/133785234/f1d1f248-b1a1-4e0a-bf7f-f4259831c126

https://github.com/coqui-ai/TTS/assets/133785234/96e5451c-d06d-4e34-aaf4-795ba8cb96e3

Feel free to also check the Tensorboard information of these runs, which also includes audio samples:

https://wandb.ai/bsc-language-tech/finetune_ca_from_en/runs/finetune_en_5e-5_5e-5_gensched-September-29-2023_10+27PM-0000000/tensorboard

Given all of that, do you have any suggestions on what we could improve?
If you need any extra information, please let us know.

Thanks in advance.

**Comment by WeberJulian** [comment]
Hey, do you mind specifying the amount of data your training on, and maybe some samples of your dataset please?

**Reply by mllopartbsc** [reply]
Hi @WeberJulian, 

Thank you for the prompt response.

First of all, here's the complete log of the training so that you can have some extra information. Before you look into it, I've got to mention that our dataset has 47 speakers.

[finetuning_en_normal_4271757.log](https://github.com/coqui-ai/TTS/files/12804535/finetuning_en_normal_4271757.log)

The amount of training data used for the fine-tuning is 22,6h. From the training log, you can calculate it with the following formula:

`(Number of instances * Avg audio length) / sampling rate `

Here are two samples of the dataset:

https://github.com/coqui-ai/TTS/assets/133785234/c722bafd-d7dd-4a20-b392-72a8336cf8a7

https://github.com/coqui-ai/TTS/assets/133785234/814f6763-e2b0-46b2-9b1b-cba435673b07

In addition, you'll also be able to find the model's training config under the 'Text' section of the Tensorboard.

Again, if you need anything else let me know.

**Comment by nellorebhanuteja** [comment]
I am having the same issue too

I am finetuning with 30 min of english data.
i am using vits vctk model as base model

i changed coqpit.lr to 0.0001 in train_vits.py
the generated voice sounds robotic, even after 1500 epochs of training

[output.wav.zip](https://github.com/coqui-ai/TTS/files/12827021/output.wav.zip)
